README.md

```diff
@@ -1 +1,55 @@
-# Hello-Microverse-1
\ No newline at end of file
+# Hello Microverse
+
+> This is the first html and css project of the first block in which linters are integrated.
+> This project is part of my Microverse exercises, where we learn how to set a up a new repository in a professional manners.
+> This project doesn't require any complex coding. The goal here is to master all of the tools and best practices that are used in a professional development environment.
+
+## Built With
+
+- HTML
+- CSS
+
+## Getting Started
+
+This is project is very simple project and it has no any further testing and deployment stages.
+
+### Prerequisites
+
+simple knowledge of HTML, CSS and git/Github is required.
+Installation of Git, installation of NPM, a working browser, live server, internet connection and a code editor like Visual Studio Code are required.
+
+### Setup
+
+Any code editor can be used, although in this instance, VS code is used.
+For cloning my repository on a local machine learn from this site- https://github.com/git-guides/git-clone
+
+### Install
+
+For setting up linters used in this project- https://github.com/microverseinc/linters-config/tree/master/html-css
+
+### Usage
+
+For opening this project on a browser. Opene the project by launching the Live Server plugging in visual studio code.
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/KenbonTN)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/kenbon-teshome/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+I am greateful for microvese community!
+
+## 📝 License
+
```

Comment:
- [ ] I notice that you are still using the old microverse README file. Please refer to [this comment by the previous reviewer](https://github.com/KenbonTN/Hello-Microverse-1/pull/1#discussion_r1067530691) on how to format the new README file. Kindly fix this and bear in mind that this is required for all microverse projects going forward.